app/models/ahoy_captain/i18n_config.rb

```diff
@@ -0,0 +1,23 @@
+module AhoyCaptain
+  class I18nConfig < ::I18n::Config
+    BACKEND = I18n::Backend::Simple.new
+    AVAILABLE_LOCALES = AhoyCaptain::Engine.root.join("config/locales/ahoy_captain").glob("*.yml").map { |path| File.basename(path, ".yml").to_sym }.uniq
```

Comment:
Maybe this is logic we want to add to the `initialize` method rather than keeping in a constant?